ONSdigital/census-rm-case-processor
# Add timestamps to UAC QID Link entity

# Motivation and Context
We needed created & updated timestamps on the UAC QID Link table, so that we can send any updated rows to DAP via the data export process, which only sends daily deltas.

# What has changed
Added new columns for created & updated to UacQidLink entity.

# How to test?
Run an acceptance test which updates a UAC QID link and make sure the updated timestamp is correctly updated, and the created timestamp remains unchanged.

# Links
Trello: https://trello.com/c/91vLJAO8
